src/actions/project.js

```diff
@@ -169,16 +170,21 @@ export function getProject(token, id) {
   }
 }
 
-export function addContributor(token, info) {
-  return async (dispatch) => {
+export function getProjectFiles(token, id) {
+  return async () => {
     try {
       const fetchClient = new FetchClient()
       fetchClient.setAuthToken(token)
-      const { result, error } = await fetchClient.post({
-        url: `/projects/${info.projectId}/contributors/${info.userId}`
+      const { result, error } = await fetchClient.get({
+        url: `/projects/${id}/zipFile`,
+        headers: {
+          'Content-Type': 'application/zip',
+          Accept: 'application/zip'
+        }
       })
+      console.log(result)
       if (result) {
-        dispatch(setProjectContributors(result))
+        download(result)
```

Comment:
You need to pass a filename in here. Otherwise, it downloads a file named "download", no zip extension or anything.

Reply:
Interesting. I was getting a file named download, but it was a zip file.